core/invoker/src/main/scala/whisk/core/container/package.scala

```diff
@@ -83,7 +83,9 @@ package object container {
         override def toString() = s"$host:$port"
     }
 
-    sealed abstract class ContainerIdentifier(val id: String)
```

Comment:
But isn't changing the `val name` in `Container` to `String` exactly the opposite of "make sure to never use `String`s when something else makes more semantic sense"?